Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -32,19 +32,6 @@ public class InputActionImporter : ScriptedImporter
         [SerializeField] private bool m_GenerateActionEvents;
         [SerializeField] private bool m_GenerateInterfaces;
 
-        // Actions and maps coming in from JSON may not have IDs assigned to them. However,
-        // once imported, we want them to have stable IDs. So we do the same thing that Unity's
-        // model importer does and remember the GUID<->name correlations used in the file.
-        [SerializeField] private RememberedGuid[] m_ActionGuids;
-        [SerializeField] private RememberedGuid[] m_ActionMapGuids;
-
-        [Serializable]
-        internal struct RememberedGuid
-        {
-            public string name;
-            public string guid;
-        }
-
```

Comment:
I nuked this mechanism as there isn't a good way to do the same for bindings. Plus the code was even wrong and it didn't work. Think overall it just added complication for something that is niche functionality.